### DISCUSSION TITLE
Per project breakpoints

### DISCUSSION THREAD

**Discussion by cshjsc** [mfussenegger/nvim-dap#582] [General]
Hi, I don't know if something like this is already possible. But every time that I close a buffer, the breakpoints disappear, it would be really cool to have breakpoints per project like Harpoon saves marks for files.

**Comment by mfussenegger** [comment]
If you enable `hidden` the breakpoints won't disappear if you close a buffer.

There are also some snippets in https://github.com/mfussenegger/nvim-dap/issues/198 to persist breakpoints to disk

**Reply by cshjsc** [reply]
Thanks
